README.md

```diff
@@ -214,15 +205,20 @@ Then refactor the application to use path-based functions rather than the defaul
 | `Router.currentUrl()` | `Router.currentPath()`  |
 | `Router.format()`     | `Router.formatPath()`   |
 | `Router.navigate()`   | `Router.navigatePath()` |
+| `Cmd.currentUrl()`    | `Cmd.currentPath()`     |
```

Comment:
@Shmew Correct me if I am wrong: I don't see why `Cmd.currentUrl()` needs to be in the commands 🤔 I also can't find the implementation of it here

Reply:
They can use `Router.current()` if they want, but since it's not pure I figured it should be possible to use via Cmd. It's located [here](https://github.com/Shmew/Feliz.Router/blob/refactor/src/Router.fs#L924).

Reply:
Can you please remove it? There is no added value to treating it as "pure", it is just unnecessary friction. (The same way we don't make a 'Math.random()` into a command itself)

Reply:
Sure, I can remove it. I do think there is value as if you try to test an update function that uses it, you will be forced to mock the function. I know if I needed `System.Random()` in an update function, I'd personally do it via `Cmd` for this reason.